client/stack-editor/lib/styl/stackwizard.styl

```diff
@@ -183,70 +177,59 @@ $accentColor                = #F2778A
       span
         color               #A8A1A1
 
-  .skip-setup
-    color                   #A8A1A1
-    font-size               16px
-    line-height             22px
-    margin-top              16px
-    text-align              center
-    font-weight             300
-    pointer()
-
   .box-wrapper
     width                   525px
     margin                  57px auto
 
-    &.services
-      margin                28px 0 0 39px !important
-
   .box
     size                    256px, 112px
+    width                   256px !important
+    height                  112px !important
     rounded                 4px
     border                  1px solid #E7E7E7
     background              #fff
+    background-position     center
+    background-repeat       no-repeat
     borderBox()
-    fl()
     pointer()
-
-    img
-      margin-top            23px
+    fl()
 
     &:nth-child(2n -1)
       margin-right          11px
 
-    .label
-      font-size             16px
-      text-align            center
-      font-weight           300
-      margin-top            13px
-
-  .services
-    .box
-      size                  227px, 95px
+    &.aws
+      r-sprite              'stack-editor' 'aws'
+    &.digitalocean
+      r-sprite              'stack-editor' 'digitalocean'
+    &.google
+      r-sprite              'stack-editor' 'googlecloud'
+    &.vagrant
+      r-sprite              'stack-editor' 'vagrant'
+    &.azure
+      r-sprite              'stack-editor' 'azure'
+    &.rackspace
+      r-sprite              'stack-editor' 'rackspace'
+    &.softlayer
+      r-sprite              'stack-editor' 'softlayer'
+
+  .beta:before
+  .coming-soon:before
+    content                 'BETA'
```

Comment:
for the localization purposes you can use html data attributes in `content` such as `content: attr(data-beta);`